database/src/main/runs/checkpoint_measure_definitions.ddl

```diff
@@ -13,20 +13,18 @@
  * limitations under the License.
  */
 
--- DROP TABLE IF EXISTS runs.checkpoint_measure_definitions;
-
 CREATE TABLE runs.checkpoint_measure_definitions
 (
     id_checkpoint_measure_definition    BIGINT NOT NULL DEFAULT global_id(),
-    key_segmentation                    BIGINT NOT NULL,
-    measure_type                        TEXT NOT NULL,
-    measure_fields                      TEXT[] NOT NULL,
+    fk_partitioning                     BIGINT NOT NULL,
+    function_name                       TEXT NOT NULL,
```

Comment:
`measure_name` is also good, what do you think?

Reply:
I wanted to have it the same as in DTO.